### PR TITLE
Python_Debugger: add 2 commands and remove trailing spaces

### PR DESCRIPTION
**Pull request by XDcedar**

### CODE REVIEW

File: cheatsheets/Python_Debugger.rb [hunk 1]
```diff
@@ -9,7 +9,12 @@
 
         entry do
             name 'Start pdb inside a python script'
-            command 'import pdb;pdb.set_trace() '
+            command 'import pdb;pdb.set_trace()'
+        end
+
+        entry do
+            name 'Same as `import pdb;pdb.set_trace()` (New in Python 3.7)'
+            command 'breakpoint()'
```
Comment (Kapeli) [inline cheatsheets/Python_Debugger.rb:17]: You can add this as a second command in the other entry. No need to create a new entry for it.

Comment (Kapeli) [reply cheatsheets/Python_Debugger.rb:17]: I'll make this change.

**Comment by Kapeli** [comment]
Thank you! 👍

File: cheatsheets/Python_Debugger.rb (hunk 1 above)
Comment (XDcedar) [reply cheatsheets/Python_Debugger.rb:17]: Yes, this is also good.

**Pull request merged**